pkg/server/controllers.go

```diff
@@ -613,23 +661,54 @@ func (s *Server) installAPIBindingController(ctx context.Context, config *rest.C
 		}
 
 		go c.Start(goContext(hookContext), 2)
-		go permissionClaimLabelController.Start(goContext(hookContext), 5)
-		go permissionClaimLabelResourceController.Start(goContext(hookContext), 2)
 
 		return nil
 	}); err != nil {
 		return err
 	}
 
+	server.AddPostStartHook(postStartHookName(permissionclaimlabel.ControllerName), func(hookContext genericapiserver.PostStartHookContext) error {
+		logger := klog.FromContext(ctx).WithValues("postStartHook", postStartHookName(permissionclaimlabel.ControllerName))
+		if err := s.waitForSync(hookContext.StopCh); err != nil {
+			logger.Error(err, "failed to finish post-start-hook")
+			return nil // don't klog.Fatal. This only happens when context is cancelled.
+		}
+
+		go permissionClaimLabelController.Start(goContext(hookContext), 5)
+
+		return nil
+	})
+
+	server.AddPostStartHook(postStartHookName(permissionclaimlabel.ResourceControllerName), func(hookContext genericapiserver.PostStartHookContext) error {
+		logger := klog.FromContext(ctx).WithValues("postStartHook", postStartHookName(permissionclaimlabel.ResourceControllerName))
+		if err := s.waitForSync(hookContext.StopCh); err != nil {
+			logger.Error(err, "failed to finish post-start-hook")
+			return nil // don't klog.Fatal. This only happens when context is cancelled.
+		}
+		go permissionClaimLabelResourceController.Start(goContext(hookContext), 2)
+
+		return nil
+	})
+
+	config = rest.CopyConfig(config)
```

Comment:
Same question as above about continuing to copy for new configs instead of copying from an unmodified config param (?)